platform/security/common/pom.xml

```diff
@@ -96,25 +110,21 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.apache.cxf</groupId>
-            <artifactId>cxf-rt-frontend-jaxws</artifactId>
-        </dependency>
-        <dependency>
             <groupId>org.powermock</groupId>
             <artifactId>powermock-module-junit4</artifactId>
-            <version>${powermock.version}</version>
+            <version>1.6.1</version>
```

Comment:
Part of Aaron's ticket (DDF-1450). If his ticket goes in first, I'll update this. If mine goes in first, I'll ask him to update his.